client/my-sites/customer-home/cards/tasks/wp-courses/index.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Task from 'calypso/my-sites/customer-home/cards/tasks/task';
+import { TASK_WP_COURSES } from 'calypso/my-sites/customer-home/cards/constants';
+import coursesLogo from 'calypso/assets/images/customer-home/courses-logo.png';
+
+const WPCourses = () => {
+	return (
+		<Task
+			title="Introducing WordPress.com Courses"
+			description="Build your skills with in-depth courses for any level, taught by WordPress experts. Enroll to access on-demand content, a private online community, office hours, and virtual meetups."
+			actionText="Enroll today!"
```

Comment:
Are these intentionally not translatable because the card is only available to English-speaking users?

Reply:
Yep, and the courses are in English so I don't anticipate these being shown to non-EN users.